packages/ui/components/dialog/Dialog.tsx

```diff
@@ -77,14 +77,14 @@ export const DialogContent = React.forwardRef<HTMLDivElement, DialogContentProps
         <DialogPrimitive.Content
           {...props}
           className={classNames(
-            "fadeIn fixed left-1/2 top-1/2 z-50 min-w-[360px] -translate-x-1/2 -translate-y-1/2 rounded bg-white text-left shadow-xl focus-visible:outline-none sm:w-full sm:align-middle",
+            "fadeIn fixed left-1/2 top-1/2 z-50 min-w-[360px] -translate-x-1/2 -translate-y-1/2 rounded-lg bg-white p-10 text-left shadow-xl focus-visible:outline-none sm:w-full sm:align-middle",
```

Comment:
@sean-brydon we added the padding directly here but maybe we should make changes only for the Embed's dialog component

Reply:
Apologies for the slow reply - been pretty ill

Missed this but! I still believe this should be p-8? p-10 would be 2.5rem -> 40px